sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -120,35 +165,48 @@ case class CatalogTablePartition(
     parameters: Map[String, String] = Map.empty,
     createTime: Long = System.currentTimeMillis,
     lastAccessTime: Long = -1,
-    stats: Option[CatalogStatistics] = None) {
+    stats: Option[CatalogStatistics] = None) extends CatalogSerializer {
+  /** Util providing the JSON-compatible version of toLinkedHashMap, for serialization of fields
+   * to JSON format via json4s
```

Comment:
I think the method name and return type says everything, we don't need this comment to repeat it.